Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -408,6 +426,48 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         updateNavigationTitle()
     }
 
+    open func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        let typingMembersWithoutCurrentUser = typingMembers
+            .sorted(
+                by: {
+                    guard let firstMemberName = $0.name, let secondMemberName = $1.name else { return false }
+                    return firstMemberName < secondMemberName
+                }
+            )
+            .filter { $0.id != self.channelController.client.currentUserId }
+        
+        if typingMembersWithoutCurrentUser.isEmpty {
+            hideTypingIndicator()
+        } else {
+            showTypingIndicator(typingMembers: typingMembersWithoutCurrentUser)
+        }
+    }
+    
+    /// Shows typing Indicator
+    /// Override this method to implement your custom logic and animations for showingthe typing indicator.
+    /// - Parameter typingMembers: typing members gotten from
```

Comment:
```suggestion
    /// Shows typing Indicator
    /// - Parameter typingMembers: typing members gotten from
```
No need to describe that the customer can customize this method, it is open, so the customer will understand it can override it. In every `open` method we are not describing that it is overridable so let's keep it consistent 👍